service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -233,11 +149,20 @@ public StepResult doStep(FlightContext flightContext)
                         new OutputDestinationRecordUpdate()
                             .recordAttribute("n_failed_chunks")
                             .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // define the WDS record to link to for inputs and outputs
             .wdsRecords(
                 new WdsRecordSet()
                     .recordType(pipelineName.getValue())
                     .addRecordIdsItem(flightContext.getFlightId()));
 
+    // add inputs
+    List<WorkflowInputDefinition> cbasWorkflowInputDefinitions =
+        pipelinesService.prepareCbasWorkflowInputRecordLookupDefinitions(
+            allInputDefinitions, wdlMethodName);
+    for (WorkflowInputDefinition workflowInputDefinition : cbasWorkflowInputDefinitions) {
```

Comment:
this `for` block is not covered by tests. think it's worth refactoring all this into a `prepareRunSetRequest` method so we can test this?